packages/bootstrap-4/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -23,8 +21,14 @@ const FieldTemplate = ({
   uiSchema,
   registry,
 }: FieldTemplateProps) => {
+  const uiOptions = getUiOptions(uiSchema);
+  const WrapIfAdditionalTemplate = getTemplate(
```

Comment:
Adding the string name as the generic will ensure the proper typing
```suggestion
  const WrapIfAdditionalTemplate = getTemplate<"WrapIfAdditionalTemplate">(
```

Reply:
I'm getting the correct type for `WrapIfAdditionalTemplate` without it. I think the generic type is inferred from the first argument, since it's a constant here.

Reply:
Hmmm, interesting. If you wanted to be consistent with the other uses of `getTemplate()` in the typescript packages consider adding it